src/sentry/api/bases/organization_events.py

```diff
@@ -108,15 +107,6 @@ def get_snuba_query_args(self, request, organization, params):
         if orderby and snuba.valid_orderby(orderby, SPECIAL_FIELDS) and 'orderby' not in snuba_args:
             snuba_args['orderby'] = orderby
 
-        # TODO(lb): remove once boolean search is fully functional
-        has_boolean_op_flag = features.has(
-            'organizations:boolean-search',
-            organization,
-            actor=request.user
-        )
-        if snuba_args.pop('has_boolean_terms', False) and not has_boolean_op_flag:
-            raise OrganizationEventsError(
-                'Boolean search operator OR and AND not allowed in this search.')
```

Comment:
Great to see this shipping :clap:

Reply:
Haha, i just removed the error and didn't ship anything. I'm not confident about it enough to ship the feature yet and I don't think it makes any sense to ship the error before the feature. But let's get a status update on boolean search before deciding what to do here.

Reply:
Ok. So it seems boolean search is dependent on CDC and doesn't sound like there is a current timeline for shipping this. I've removed the error (imo there's no value in shipping the error before the feature), and commented out the line that handles boolean search in `events_search`